lib/sycamore/sycamore/evaluation/evaluate.py

```diff
@@ -0,0 +1,183 @@
+from typing import Dict, List, Union, Any
+from abc import ABC, abstractmethod
+
+import os
+from sycamore.data import Element, Document
+from sycamore.evaluation.pipeline import EvaluationPipeline
+from sycamore.evaluation import EvaluationDataPoint, EvaluationMetric
+from sycamore.evaluation.metrics import document_retrieval_metrics
+from sycamore import Context
+
+
+def add_search_context_to_datapoint(datapoint: Document) -> List[Element]:
+    """
+    This function adds raw inforation from json ground truth and properties
+    to each evaluation datapoint
```

Comment:
Minor nit (while we are here!) -- it is best for docstrings to be complete sentences, i.e., ending with a period, so it is more readable and it's clear to a reviewer that something did not get cut off by accident.

Reply:
thanks, made changes to complete sentences.